### DISCUSSION TITLE
Can't plot when define terminal_states()

### DISCUSSION THREAD

**Discussion by alexandre-jf-fernandes** [theislab/cellrank#1089] [Method/Algorithm]
Hi there,

I'm trying to get a heatmap of genes of each macrostate, but when run `g_fwd.compute_terminal_states()
g_fwd.compute_absorption_probabilities()` the not terminal lineages can't longer be used for plotting the heatmap or calculating lineage_drivers. Can you help me?

`cr.pl.heatmap(adata,
                   model,
                   genes=epi_list_filtered,
                   backend='loky', 
                   show_all_genes=True,
                   show_absorption_probabilities=True,
                   time_key='velocity_pseudotime',
                   cluster_genes=True,
                   cmap='viridis',
)`

Output: Just terminal state lineage

**Comment by Marius1311** [comment]
Please open an issue for this. For discussion topics (more general questions, not a bug or feature request), see https://discourse.scverse.org/c/ecosystem/cellrank/40
